docs/overview/analysis/plots.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+const BokehFigure = React.memo(({ pathToData }) => {
+  const plotData = React.useMemo(() => require(`${pathToData}`), []);
+  const targetID = plotData["target_id"];
+  return (
+    <div className="bk-root" id={targetID}>
+      <BrowserOnly fallback={<img src={useBaseUrl(`/img/${targetID}.png`)} />}>
+        {() => {
+          {
+            window.Bokeh.embed.embed_item(plotData, targetID);
+          }
+        }}
+      </BrowserOnly>
+    </div>
+  );
+});
```

Comment:
Yes, and this is done in #1278 where we have the reusable React components in the `website/src/components` directory. It would be nice to get that PR in first, and then have me merge it into this PR, but that all depends on the priority of the PRs.

Reply:
Ah, you're right. I forgot that the tutorials also use the same Bokeh component for interactive plots. I'll take a look at the other PR soon. Thanks!

Reply:
These duplicate code block have now been removed.